tests/integrated/test-schedule.R

```diff
@@ -80,9 +80,10 @@ test_that("schedule display works", {
   expect_snapshot_output(schedule_describe(tmp))
 })
 
+# Doesn't work in integration tests for some reason.
```

Comment:
Then delete this? No good leaving commented code around. 

This should be solvable I would think, set something in the Docker image and assert we're picking it up here. IDK that it's worth the effort for such an insignificant API though.

Reply:
Will do.